lib/sms.rb

```diff
@@ -0,0 +1,22 @@
+require 'twilio-ruby'
+
+class Sms
+
+  def initialize
+    account_sid = 'twilio acc sid'
+    auth_token = 'twilio auth token'
+    @client = Twilio::REST::Client.new account_sid, auth_token
```

Comment:
A call to .new within a method like this is a bit of a red flag.  It means that you have a very hard dependency  on another class.  This is a problem for at least two reasons:

1) Your class can only ever work with the class it is depending on.  You have no flexibility and it would be very difficult to reuse your code in a different situation.
2) It's very difficult to properly isolate your tests.

The 2nd point is particularly important here - if your tests aren't isolated from using a real version of the Twilio client you'll be getting a text message every time you run RSpec.